config/core/build-configs-stable.yaml

```diff
@@ -83,6 +86,26 @@ stable_variants_kselftest: &stable_variants_kselftest
 
 build_configs:
 
+  cip_stable_4.4:
+    tree: cip
+    branch: 'linux-4.4.y-st'
+    variants: *stable_variants
+
+  cip_stable_4.4-rc:
+    tree: cip
+    branch: 'linux-4.4.y-st-rc'
+    variants: *stable_variants
+
+  cip_stable_4.4-rt:
+    tree: cip
+    branch: 'linux-4.4.y-st-rt'
```

Comment:
there is no `linux-4.4.y-st-rt` -> but there is `linux-4.4.y-st-rc`

Reply:
is that branch actually correct? maybe upstream should rename it to rt if is a typo

Reply:
My understanding is that there _should_ be an `linux-4.4.y-st-rt` branch, but at the moment there isn't.

Reply:
probably is a typo made by pavel ?

Reply:
pavel added `linux-4.4.y-st-rt` branch to kernel.org